photutils/psf/photometry.py

```diff
@@ -0,0 +1,448 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+"""Module which provides classes to perform PSF Photometry"""
+
+from __future__ import division
+import numpy as np
+from astropy.table import Table, vstack, hstack
+from astropy.modeling.fitting import LevMarLSQFitter
+from astropy.nddata.utils import overlap_slices
+from astropy.stats import gaussian_sigma_to_fwhm
+from ..psf import subtract_psf
+from ..aperture import CircularAperture, aperture_photometry
+
+try:
+    from astropy.nddata.utils import NoOverlapError
```

Comment:
We've just talked about it, move this import to the place it's used, so the importerror is not coming up at import time with older astropies. Maybe leave a note as well, that this can be cleaned up once stop supporting 1.0.x LTS